### ISSUE TITLE
Use regexp in routes?

### ISSUE THREAD

**Issue by elcobvg** [sveltejs/sapper#274]
We've decided to use sapper for the new version of a multi lingual website. Obviously, there's a few challenges left to achieve proper i18n. I've already found some useful pointers, but I have a problem with the URL structure which has to stay the same as the current site: all the languages have a locale prefix, except the default language. So you get something like this:
```
/fr/[category]/[page].html
/de/[category]/[page].html
/nl/[category]/[page].html
/[category]/[page].html
```
I found a neat solution for the languages here #78, using a route like `/[lang]/[category]/[page].html`, but that will conflict with my default locale. It would be very useful if we could use a regexp in the route like so: `[/fr|de|nl/]/[category]/[page].html`
Or is there perhaps another smart solution that I'm missing?

**Comment by Rich-Harris** [comment]
This can be closed now, was released in 0.14.1 (thanks @elcobvg!)

**Comment by elcobvg** [comment]
My pleasure :)
